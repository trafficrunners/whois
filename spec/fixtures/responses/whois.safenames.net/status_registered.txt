Whois Server Version 2.0

Domain names in the .com and .net domains can now be registered
with many different competing registrars. Go to http://www.internic.net
for detailed information.

   Domain Name: STRIPE.COM
   Registrar: SAFENAMES LTD
   Whois Server: whois.safenames.net
   Referral URL: http://www.safenames.net
   Name Server: NS-1087.AWSDNS-07.ORG
   Name Server: NS-1882.AWSDNS-43.CO.UK
   Name Server: NS-423.AWSDNS-52.COM
   Name Server: NS-705.AWSDNS-24.NET
   Status: clientDeleteProhibited
   Status: clientTransferProhibited
   Status: clientUpdateProhibited
   Status: serverDeleteProhibited
   Status: serverTransferProhibited
   Status: serverUpdateProhibited
   Updated Date: 02-oct-2014
   Creation Date: 12-sep-1995
   Expiration Date: 11-sep-2024

>>> Last update of whois database: Mon, 29 Dec 2014 19:01:25 GMT <<<

NOTICE: The expiration date displayed in this record is the date the
registrar's sponsorship of the domain name registration in the registry is
currently set to expire. This date does not necessarily reflect the expiration
date of the domain name registrant's agreement with the sponsoring
registrar.  Users may consult the sponsoring registrar's Whois database to
view the registrar's reported date of expiration for this registration.

TERMS OF USE: You are not authorized to access or query our Whois
database through the use of electronic processes that are high-volume and
automated except as reasonably necessary to register domain names or
modify existing registrations; the Data in VeriSign Global Registry
Services' ("VeriSign") Whois database is provided by VeriSign for
information purposes only, and to assist persons in obtaining information
about or related to a domain name registration record. VeriSign does not
guarantee its accuracy. By submitting a Whois query, you agree to abide
by the following terms of use: You agree that you may use this Data only
for lawful purposes and that under no circumstances will you use this Data
to: (1) allow, enable, or otherwise support the transmission of mass
unsolicited, commercial advertising or solicitations via e-mail, telephone,
or facsimile; or (2) enable high volume, automated, electronic processes
that apply to VeriSign (or its computer systems). The compilation,
repackaging, dissemination or other use of this Data is expressly
prohibited without the prior written consent of VeriSign. You agree not to
use electronic processes that are automated and high-volume to access or
query the Whois database except as reasonably necessary to register
domain names or modify existing registrations. VeriSign reserves the right
to restrict your access to the Whois database in its sole discretion to ensure
operational stability.  VeriSign may restrict or terminate your access to the
Whois database for failure to abide by these terms of use. VeriSign
reserves the right to modify these terms at any time.

The Registry database contains ONLY .COM, .NET, .EDU domains and
Registrars.
Domain Name: STRIPE.COM
Registrar WHOIS Server: whois.safenames.net
Registrar URL: http://www.safenames.net
Updated Date: 2014-10-02T15:33:46Z
Created Date: 1995-09-12T04:00:00Z
Registrar Registration Expiration Date: 2024-09-11T04:00:00Z
Registrar: Safenames Ltd
Registrar IANA ID: 447
Registrar Abuse Contact Email: abuse@safenames.net
Registrar Abuse Contact Phone: +44.1908200022
Registrant Name: Domain Admin
Registrant Organisation: Stripe
Registrant Address Line 1: 3180 18th St
Registrant Address Line 2:
Registrant City: San Francisco
Registrant State/Province: CA
Registrant Postal Code: 94110
Registrant Country: US
Registrant Phone: +1.8772544179
Registrant Fax:
Registrant Email: dns@stripe.com
Admin Name: Domain Admin
Admin Organisation: Stripe
Admin Address Line 1: 3180 18th St
Admin Address Line 2:
Admin City: San Francisco
Admin State/Province: CA
Admin Postal Code: 94110
Admin Country: US
Admin Phone: +1.8772544179
Admin Fax:
Admin Email: dns@stripe.com
Tech Name: Domain Admin
Tech Organisation: Stripe
Tech Address Line 1: 3180 18th St
Tech Address Line 2:
Tech City: San Francisco
Tech State/Province: CA
Tech Postal Code: 94110
Tech Country: US
Tech Phone: +1.8772544179
Tech Fax:
Tech Email: dns@stripe.com
Name Server: dns1.idp365.net
Name Server: dns2.idp365.net
URL of the ICANN WHOIS Data Problem Reporting System: http://wdprs.internic.net/

Safenames - Experts in Global Domain Management and Online Brand Protection

Domain Registration in over 760 different extensions
Enterprise Domain Management since 1999
Mark Protectâ„¢ Online Brand Monitoring and Enforcement
Domain Consulting and Strategy
Domain Name Acquisition
Domain Disputes and Recovery

Visit Safenames at www.safenames.net
+1 703 574 5313 in the US/Canada
+44 1908 200022 in Europe

The Data in the Safenames Registrar WHOIS database is provided by Safenames for
information purposes only, and to assist persons in obtaining information about
or related to a domain name registration record.  Safenames does not guarantee
its accuracy.  Additionally, the data may not reflect updates to billing
contact information.

By submitting a WHOIS query, you agree to use this Data only for lawful purposes
and that under no circumstances will you use this Data to:

(1) allow, enable, or otherwise support the transmission of mass unsolicited,
commercial advertising or solicitations via e-mail, telephone, or facsimile; or
(2) enable high volume, automated, electronic processes that apply to Safenames
(or its computer systems).  The compilation, repackaging, dissemination or
other use of this Data is expressly prohibited without the prior written
consent of Safenames.  Safenames reserves the right to terminate your access to
the Safenames Registrar WHOIS database in its sole discretion, including
without limitation, for excessive querying of the WHOIS database or for failure
to otherwise abide by this policy.  Safenames reserves the right to modify
these terms at any time. By submitting this query, you agree to abide by this
policy.
