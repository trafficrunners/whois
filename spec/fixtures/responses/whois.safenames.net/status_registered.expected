#domain
  %s == "stripe.com"

#domain_id
  %s == nil


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{1995-09-12T04:00:00Z}

#updated_on
  %s %CLASS{time}
  %s %TIME{2014-10-02T15:33:46Z}

#expires_on
  %s %CLASS{time}
  %s %TIME{2024-09-11T04:00:00Z}


#registrar
  %s %CLASS{registrar}
  %s.id           == "447"
  %s.name         == "Safenames Ltd"
  %s.organization == "Safenames Ltd"
  %s.url          == "http://www.safenames.net"


#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == nil
  %s[0].name          == "Domain Admin"
  %s[0].organization  == "Stripe"
  %s[0].address       == "3180 18th St"
  %s[0].city          == "San Francisco"
  %s[0].zip           == "94110"
  %s[0].state         == "CA"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.8772544179"
  %s[0].fax           == ""
  %s[0].email         == "dns@stripe.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == nil
  %s[0].name          == "Domain Admin"
  %s[0].organization  == "Stripe"
  %s[0].address       == "3180 18th St"
  %s[0].city          == "San Francisco"
  %s[0].zip           == "94110"
  %s[0].state         == "CA"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.8772544179"
  %s[0].fax           == ""
  %s[0].email         == "dns@stripe.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[0].id            == nil
  %s[0].name          == "Domain Admin"
  %s[0].organization  == "Stripe"
  %s[0].address       == "3180 18th St"
  %s[0].city          == "San Francisco"
  %s[0].zip           == "94110"
  %s[0].state         == "CA"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.8772544179"
  %s[0].fax           == ""
  %s[0].email         == "dns@stripe.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "dns1.idp365.net"
  %s[1] %CLASS{nameserver}
  %s[1].name == "dns2.idp365.net"
